libs/features/mas/web-components/test/merch-card.catalog.test.html.js

```diff
@@ -0,0 +1,71 @@
+// @ts-nocheck
+import { runTests } from '@web/test-runner-mocha';
+import { expect } from '@esm-bundle/chai';
+
+import { mockLana } from './mocks/lana.js';
+import { mockFetch } from './mocks/fetch.js';
+import { mockConfig } from './mocks/config.js';
+
+import '../src/merch-offer.js';
+import '../src/merch-offer-select.js';
+import '../src/merch-quantity-select.js';
+
+import { appendMiloStyles, delay } from './utils.js';
+import { mockIms } from './mocks/ims.js';
+import { withWcs } from './mocks/wcs.js';
+import mas from './mas.js';
+
+const skipTests = sessionStorage.getItem('skipTests');
+
+runTests(async () => {
+    mockIms();
+    mockLana();
+    await mockFetch(withWcs);
+    await mas();
+    if (skipTests !== null) {
+        appendMiloStyles();
```

Comment:
if you feel,  remove `appendMiloStyles`, and maybe reference the styles in question in the test page.

Reply:
finally someone will do that later, as it's more work than i was expected :)